.gitattributes

```diff
@@ -0,0 +1,62 @@
+################################################
+#  LangSandbox by Federico Tomassetti:         #
+#  https://github.com/ftomassetti/LangSandbox  #
+################################################
+
+# Set the default behavior, in case Win users didn't set core.autocrlf.
+* text=auto
+
+
+# Explicilty declare EOL normalization and binary files for all file types
+# used in the project. ("Explicit is better than implicit." PEP 20 Zen)
+
+
+## Our New Languages
+####################
+*.sandy   text
+
+
+## Repository Configurations
+############################
+.editorconfig    text eol=lf
+.gitlab-ci.yml   text eol=lf
+.travis.yml      text eol=lf
+.gitattributes   text eol=lf
+.gitconfig       text eol=lf
+.gitignore       text eol=lf
+.gitmodules      text eol=lf
+
+
+## Shell Scripts
+################
+gradlew   text eol=lf
+*.bat     text eol=crlf
+*.cmd     text eol=crlf
+*.sh      text eol=lf
+*.ps1     text eol=crlf
```

Comment:
Do we need ps1 files?

Reply:
I've added it to be on the safe side — you never know, we might soon need them if some third party tools make switch from batch files to PowerShell (e.g. Gradle), which would make sense due to the limitations of batch scripts today, whereas PS is more powerful, especially if there's a need to interact with WSL/WSL2 (which adds native support for Linux binaries). I think that ultimately many third party tools will switch to PS scripts (Chocolatey, to name one, uses PS). 

Also, when it comes to Git scripts, PS offers better interaction, so it can be useful for complex automation tasks (e.g. on Travis CI using a Win Server VM).

If I had batch scripts I also add PS scripts, after all it's a one-liner which doesn't harm anyone if unused, whereas if these are lacking it might be a huge problem if non-Windows users edit those files and introduce `LF` EOLs.

Of course, we can delete the line if you want to.

Reply:
Makes sense, let's keep it